pkgs/applications/misc/lttoolbox/default.nix

```
@@ -0,0 +1,45 @@
+{ fetchFromGitHub, lib, pkgs, stdenv }:
```

Using `pkgs` in nixpkgs is generally discouraged.

```suggestion
{ lib
, stdenv
, fetchFromGitHub
, autoconf
, automake
, pkg-config
, utf8cpp
, libtool
, libxml2
, icu
}:
```